index.js

```diff
@@ -0,0 +1,9 @@
+//Exception handling
+
+try {
+  let a = 3 / 0;
```

Comment:
In JS we can divide by zero. You need to throw your own custom error if result of division is equal to `Infinity`. Also I cannot see a solution for first part of this task.